packages/mobile/App.svelte

```diff
@@ -118,6 +119,7 @@
         }, 3000)
 
         initRouters()
+        await getVersionDetails()
```

Comment:
Why do we need this here? `packages/shared/components/popups/Diagnostics.svelte` already does 
```
void Platform.getDiagnostics().then((values) => (contentSystem = combineValues(values)))
```
wouldnt that be enough?

Reply:
On CapacitorApi.ts we don't have implemented `getDiagnostics()` yet, on desktop it's needed to get the OS version details and hardware features. We could implement it in the future to get hardware features on mobile too using `@capacitor/device`. 
To show the app version we need only to call `getVersionDetails()` that is calling `Platform.getVersionDetails()` just implemented.

Reply:
ok implemented 😄 we get this info now:
```
{ label: 'Name', value: info.name },
{ label: 'Model', value: info.model },
{ label: 'OS', value: info.operatingSystem },
{ label: 'OS version', value: info.osVersion },
{ label: 'Manufacturer', value: info.manufacturer },
{ label: 'Webview version', value: info.webViewVersion },
{ label: 'Memory used', value: `${info.memUsed / 1024} MB` },
{ label: 'Disk free', value: `${info.realDiskFree / 1024} MB` },
```
very useful for debug and fix issues, thanks for the appointment!

Reply:
Don't know if I get the question right but as far as I can see `Platform.getDiagnostics` doesn't deliver the version in Diagnostics.svelte, but the versionDetails store does some lines above. To see the version, the store has to be filled with `getVersionDetails` in App.svelte. Maybe the function name is a bit irritating, because it actually sets the version and does not return one.

Reply:
exactly you've explained very nice it 💯